edgar-doctor/src/components/app/dashboardPages/patients/subViews/PatientsSubViewAppointments.tsx

```diff
@@ -0,0 +1,88 @@
+'use client';
+
+import { useState } from 'react';
+
+import { Button, HStack, VStack } from '@chakra-ui/react';
+
+import paginationHandler from 'utils/navigation/paginationHandler';
+import Pagination from 'components/navigation/Pagination';
+import countMaxNumberPage from 'utils/navigation/countMaxNumberPage';
+
+import { type PatientType } from 'types/app/dashboard/patients/PatientType';
+import { useGetDoctorAppointmentsQuery } from 'services/request/appointments';
+import AppointmentCard from '../../appointments/AppointmentCard';
+
+const PatientsSubViewAppointments = ({
+	selectedPatient,
+}: {
+	selectedPatient: PatientType | undefined;
+}): JSX.Element => {
+	const [pageIndexNext, setPageIndexNext] = useState(1);
+	const [pageIndexOld, setPageIndexOld] = useState(1);
```

Comment:
Rename these two states to avoid misunderstanding => follow the same pattern as the appointment types